### PR TITLE
Project Documentation-SmartBot Navigator

### PR DESCRIPTION
**Pull request by Raluca3012**
### Pull Request Overview

This pull request adds/changes/fixes...

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Your Name <Your Email@example.com>

### CODE REVIEW

File: docs/project/2024/mihaela_raluca.dorobantu/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# SmartBot Navigator
+An autonomous indoor robot equipped with a 180-degree rotating ultrasonic sensor, powered by a Raspberry Pi Pico, and transmitting real-time data via Bluetooth.
+
+
+:::info 
+
+**Author**: Mihaela-Raluca Dorobanțu
```
Comment (alexandruradovici) [inline docs/project/2024/mihaela_raluca.dorobantu/index.md:7]:
```suggestion
**Author**: Mihaela-Raluca Dorobanțu \
```

File: docs/project/2024/mihaela_raluca.dorobantu/index.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+| [Source (for Raspberry Pi Pico) + Breadbord + Wires] | Provides power to the microcontroller  | [35 RON](https://www.emag.ro/kit-breadboard-830-gauri-65-fire-modul-tensiune-alimentare-mb102-tri273/pd/DY1YP6BBM/)|
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/mihaela_raluca.dorobantu/index.md:107]: Do you need these libraries?

File: docs/project/2024/mihaela_raluca.dorobantu/index.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://example.com)
+2. [link](https://example3.com)
```
Comment (alexandruradovici) [inline docs/project/2024/mihaela_raluca.dorobantu/index.md:114]: Please add some links.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template?

File: docs/project/2024/mihaela_raluca.dorobantu/index.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+| [Battery Pack (for Motor Controller)](https://) | Supplies energy to the motors and servo | [3 RON](https://www.emag.ro/suport-cu-fire-pentru-baterii-9v-oky0252-1/pd/D6R726MBM/) |
+| [Source (for Raspberry Pi Pico) + Breadbord + Wires](https://) | Provides power to the microcontroller  | [35 RON](https://www.emag.ro/kit-breadboard-830-gauri-65-fire-modul-tensiune-alimentare-mb102-tri273/pd/DY1YP6BBM/) |
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/mihaela_raluca.dorobantu/index.md:106]: Do you need this library?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template.

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**